sample_code/ep436/README.md

```diff
@@ -0,0 +1,47 @@
+# [a git workflow for *only my branches* (intermediate)](https://youtu.be/GKBq5Xo_B6I)
+
+Today I show off a git workflow I use when I'm working on a repository with hundreds or thousands of other branches (such as at work!)
+
+## Interactive examples
+
+### Bash
+
+```bash
+git clone git@github.com:getsentry/sentry
+cd sentry/
+
+git branch -r
+git branch -r | wc -l
+git branch -r | grep asottile
+
+git fetch
+babi .git/config
+
+# https://github.com/asottile/scratch/wiki/only-track-my-branches
```

Comment:
Not sure if you intentionally omitted the link in the YT description or you just forgot, so I've added a comment here.

Reply:
oops I just forgot, will fix